src/Administration/Resources/app/administration/src/app/init/locale.init.ts

```
@@ -1,21 +1,19 @@
 /**
  * @sw-package framework
  */
-
 // eslint-disable-next-line sw-deprecation-rules/private-feature-declarations
 export default async function initializeLocaleService() {
     const factoryContainer = Shopware.Application.getContainer('factory');
     const localeFactory = factoryContainer.locale;
-
-    // Register default snippets
-    localeFactory.register('de-DE', {});
-    localeFactory.register('en-GB', {});
-
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
     const snippetService = Shopware.Service('snippetService');
 
     if (snippetService) {
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-member-access
+        const locales = await snippetService.getLocales();
```

Wouldn't it be better if a proper JS error is raised if the snippetService somehow isn't available instead of failing silently and just not registering the snippets? Or at least printing a `console.warn` could be helpful.

---

Good catch!